tests/integration/devfile/cmd_devfile_registry_test.go

```diff
@@ -81,6 +83,30 @@ var _ = Describe("odo devfile registry command tests", func() {
 			helper.CmdShouldPass("odo", "registry", "delete", registryName, "-f")
 			helper.CmdShouldFail("odo", "create", "java-maven", "--registry", registryName)
 		})
+	})
 
+	Context("when working with git based registries", func() {
+		It("should show deprecation warning when the git based registry is used", func() {
+			deprecated := "Deprecated"
+			docLink := "https://github.com/openshift/odo/tree/main/docs/public/git-registry-deprecation.adoc"
+			outstr, errstr := helper.CmdShouldPassIncludeErrStream("odo", "registry", "add", "RegistryFromGitHub", "https://github.com/odo-devfiles/registry")
+			co := fmt.Sprintln(outstr, errstr)
```

Comment:
Is it required to print this?

Reply:
It is not being printed. It is sprintf that is combining the stdout and stderr into one blob so I can do 1 comparision

Reply:
> It is not being printed.

Yes, my bad. Actually I saw this as `Println` 😄